### DISCUSSION TITLE
Set type in HTML

### DISCUSSION THREAD

**Discussion by alsted** [vakata/jstree#2489] [Q&A]
Hi 
Is there a way to set the node type in HTML?  
I can set the icons but it would be nice to set the type and let the types plugin control the icon etc.
Best

**Comment by vakata** [comment]
You can use the jstree data attribute - something like ```data-jstree='{"type":"test"}'```

**Reply by alsted** [reply]
Thanks. I missed that in the documentation. Can you point me to where I should have picked that up?
